Migrate to using @actions/core and @actions/github (#23)
dev-drprasad/delete-tag-and-release: README.md

    delete_release: true #(optional) default: true 
    repo: <owner>/<repoName> #(optional) target repository. default: repo running this action
```

## Developing

You can follow these instructions to get this project cloned to your local machine and ready for development. Note that
development requires at least [Node 16](https://nodejs.org/en/download) to be installed on your computer.

```shell
git clone https://github.com/dev-drprasad/delete-tag-and-release.git
cd delete-tag-and-release
npm install
```

Tests can be run via:

```shell
npm test
```

> Since this is a repo for a GitHub Action, **every new commit that changes code must include a new bundle.** You can 

Reviewer: ```suggestion
Since this is a repo for a [GitHub Action](https://docs.github.com/en/actions), **every new commit that changes code must include a new bundle.** You can 
```

generate a new bundle via running:

```shell
npm run package
```

> and commiting the resulting changes that will be in the `dist/` folder.

Reviewer: ```suggestion
and committing the resulting changes that will be in the `dist/` folder.
```

## Releasing

> As this repo is a GitHub Action, releases are done via creating a new tag and pushing that tag to the repo. A new

Reviewer: ```suggestion
Releases are done by creating a new tag and pushing that tag to the repo. A new
```
Let's try to use Simple English, to make this easily translatable for those that don't know English natively.

release can be made via the following:

```shell
git checkout master
git fetch
git pull
# choose either patch, minor, or major. See https://docs.npmjs.com/cli/v8/commands/npm-version#description for more info
npm version <patch/minor/major>
git push --follow-tags
```

This will update the version in the `package.json`, create a new commit based off of the master branch, and create a 

> new Git tag named the new version that points to that new commit. Remember the `--follow-tags` flag when running 

Reviewer: ```suggestion
new Git tag with that version that points to the new commit. Remember to use the `--follow-tags` flag when running 
```
